### PR TITLE
URLScan Enhancements

### PR DESCRIPTION
**Pull request by amshamah419**
1. Deprecated IP, File, and URL command since the integration does not accept them.
2. Formatted outputs to be easily accessible for customers
3. Migrated integration to Python

### CODE REVIEW

File: Integrations/integration-UrlScan.yml [hunk 1]
```diff
@@ -16,496 +15,640 @@ configuration:
...
       - public
       - private
-      description: Does the submit will be public or private
+      description: Will the submission be public or private
       defaultValue: public
+    - name: limit
+      description: Limits the results in the war room
+      defaultValue: "20"
     outputs:
-    - contextPath: URL.Data
```
Comment (Itay4) [inline Integrations/integration-UrlScan.yml]: Why did you remove all these outputs? Please add them back

File: Tests/conf.json [hunk 2]
```diff
@@ -49,10 +49,7 @@
           "playbookID": "devo_test_playbook"
         },
         {
-            "integrations": [
-                "VirusTotal",
-                "urlscan.io"
-            ],
+          "integrations": "VirusTotal",
```
Comment (Itay4) [inline Tests/conf.json]: Why are you using `VirusTotal` and no `urlscan` ?

Comment (amshamah419) [reply Tests/conf.json]: Switched to urlscan.
